### PR TITLE
Servlet setup

### PR DESCRIPTION
**Pull request by michaeleggboy**
Commits centered around the functionality of servlet setup.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlet/NewCommentServlet.java
Comment (MarsZ42) [inline portfolio/src/main/java/com/google/sps/servlet/NewCommentServlet.java]: I know it’s sometimes easier to just comment out something you need for testing things out, but remember to remove them when you trying to merge them into master. Just add them back when you need them.

File: portfolio/src/main/webapp/script.js
Comment (MarsZ42) [inline portfolio/src/main/webapp/script.js]: Rename these method to something descriptive

File: portfolio/src/main/webapp/script.js
Comment (MarsZ42) [inline portfolio/src/main/webapp/script.js:27]: It might not be a good idea to share login and out in one url. Try using different paths for them.

**Review by MarsWang42 — CHANGES_REQUESTED** [review]
Do not merge the PR before I approved.

**Comment by michaeleggboy** [comment]
Everything should be good now, ready to merge

**Review by MarsWang42 — APPROVED** [review]

**Pull request merged**
